source/guide_filebrowser.rst

```diff
@@ -0,0 +1,121 @@
+.. highlight:: console
+
+.. author:: on4r <https://github.com/on4r>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: web
+.. tag:: file-storage
+.. tag:: self-hosting
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/filebrowser.png
+      :align: center
+
+############
+File Browser
+############
+
+.. tag_list::
+
+`File Browser`_ provides a file managing interface within a specified directory and it can be used to upload, delete, preview, rename and edit your files.
+It allows the creation of multiple users and each user can have its own directory. It can be used as a standalone app or as a middleware.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+File Browser is licensed under the `Apache License 2.0`_
+
+Installation
+============
+
+Download and extract the `latest release`_
+------------------------------------------
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/bin
+ [isabell@stardust bin]$ curl --location <url_of_linux_amd64_filebrowser> | tar -xzf - filebrowser
+
+Initialize a new database
+-------------------------
+
+.. code-block:: console
+
+ [isabell@stardust bin]$ cd ~
+ [isabell@stardust ~]$ mkdir filebrowser && cd filebrowser
+ [isabell@stardust filebrowser]$ filebrowser config init
+
+
+Configuration
+=============
+
+.. note:: You have to be in the same directory as the *filebrowser.db* file which you created before, in order to configurate it or else it will create a new one.
+
+.. code-block:: console
+
+ [isabell@stardust filebrowser]$ filebrowser config set --address 0.0.0.0
+ [isabell@stardust filebrowser]$ filebrowser config set --root $HOME/my_files
+
+Create your admin account.
+
+.. code-block:: console
+
+ [isabell@stardust filebrowser]$ filebrowser users add '<username>' '<password>' --perm.admin
+
+File Browser runs on port ``8080`` by default.
+
+.. include:: includes/web-backend.rst
+
+Create your file folder (the one you specified via the ``--root`` option).
```

Comment:
better put this part at the beginning of configuration and reuse the chosen path afterwards